### ISSUE TITLE
Minor layout, styling and text improvements

### ISSUE THREAD

**Issue by johanstokking** [TheThingsNetwork/lorawan-stack#875]
#### Summary

Minor layout, style and text improvements for the Console.

#### Why do we need this?

To make it even more awesome than it already is.

#### What do you want to see?

- [x] Rename "Connected Devices" to "Devices". We can't claim they are "connected" (#888)
- [ ] Add "connected" indicator to gateway list by checking GS connection stats (#1155)
- [x] Increase page size from 5 to 10 (device list in application overview, table can be a bit more condensed) or 20 (list views of applications, devices and gateways) (#907)
- [ ] Use consistent Title Case, i.e. "Message _T_ype" in webhook config (#1273)
- [x] We need alphabetic sorting in lists by default (#1054 / #1107)
- [x] Move API key name to first column (#895)
- [x] Make terms like "TTN Stack" configurable (currently used in things like "TTN Stack Console" and "TTN Stack Account"). We have the site names for this, right?

#### Can you do this yourself and submit a Pull Request?

Can review

**Comment by bafonins** [comment]
> Increase page size from 5 to 10 (device list in application overview, table can be a bit more condensed) or 20 (list views of applications, devices and gateways)

There is the `Devices` entry in the side navigation with the list of devices (25 devices per page).

> We need alphabetic sorting in lists by default

Shouldnt this be done by IS?

**Comment by bafonins** [comment]
> Add "connected" indicator to gateway list by checking GS connection stats

and

>  Increase page size from 5 to 10 (device list in application overview, table can be a bit more condensed) or 20 (list views of applications, devices and gateways)

I can agree that showing the connection status for gateways is nice, however do we really want to make 5-20 calls to fetch the stats on every page load?

**Comment by johanstokking** [comment]
> > Increase page size from 5 to 10 (device list in application overview, table can be a bit more condensed) or 20 (list views of applications, devices and gateways)
> 
> There is the `Devices` entry in the side navigation with the list of devices (25 devices per page).

OK, but not the applications and gateways, see e.g.

<img width="731" alt="Screenshot 2019-07-01 at 14 32 44" src="https://user-images.githubusercontent.com/13334001/60436934-78667800-9c0d-11e9-8bd8-687b02e81e45.png">

> > We need alphabetic sorting in lists by default
> 
> Shouldnt this be done by IS?

Yeah it's the responsibility of where pagination happens, because we first need to sort before we can paginate. I don't think we should go as far as sorting by a selected column, but at least by one entity "primary" column (i.e. ID).

> > Add "connected" indicator to gateway list by checking GS connection stats
> 
> and
> 
> > Increase page size from 5 to 10 (device list in application overview, table can be a bit more condensed) or 20 (list views of applications, devices and gateways)
> 
> I can agree that showing the connection status for gateways is nice, however do we really want to make 5-20 calls to fetch the stats on every page change?

Yeah I think so. These calls are cheap and we could optimize it later by grouping stats check per GS.

**Comment by bafonins** [comment]
> OK, but not the applications and gateways, see e.g.

So, shall we increase page size for applications and gateways to 10? cc @kschiffer

**Comment by htdvisser** [comment]
I can look into sorting of those listings

**Comment by kschiffer** [comment]
> > OK, but not the applications and gateways, see e.g.
> 
> So, shall we increase page size for applications and gateways to 10? cc @kschiffer

Yes, I think 10 is a good default. Let's also use a unified constant for this.

**Comment by kschiffer** [comment]
> Move API key name to first column

See [my comment](https://github.com/TheThingsNetwork/lorawan-stack/pull/895#issuecomment-507593303) on improving the list display of api keys as a whole:
> The ID value is very unhelpful IMO. I don't see any user scanning the table by a cryptic hash value. If we can, we should really sort by creation date or the name value. We might even consider making the name field mandatory, or setting default values like API Key 1, which I would recommend. We can then hide or at least abbreviate the ID field in the table view (that is, if there is no essential use of the ID value that I have missed).

If there's no objection I would like to solve it this way.

**Comment by htdvisser** [comment]
The ID is the middle part of the API key, so if a user has an API key, they can see the key's rights by finding that same ID in the list.

I would propose to leave things the way they are (so sort by ID) until we have #316 which allows us to sort by fields other than ID.

**Comment by johanstokking** [comment]
@kschiffer I think 10 is too little. The Console should be informative. If it doesn't fit, consider making the table more condense. I'd say at least 20 but maybe 25 or 30 items.

**Comment by kschiffer** [comment]
Please move this to #907.

**Comment by johanstokking** [comment]
Please address also replace hardcoded `The Things Stack Account` by `{siteName} Account`

**Comment by kschiffer** [comment]
> Please address also replace hardcoded `The Things Stack Account` by `{siteName} Account`

via #1323

**Comment by kschiffer** [comment]
Closing in favor of actionable issues (see OP).
